src/translations/screens/subscreens/EditProfileScreen.ts

```diff
@@ -45,6 +45,11 @@ export const EditProfileTexts = {
         'The e-mail address is already in use',
         'E-postadressa er allereie i bruk',
       ),
+      disabledWithRemoteConfig: _(
+        'Appen støtter foreløpig ikke lagring av e-postadresse.',
+        'The app currently does not support saving your email address',
+        'Appen stettar førebels ikkje lagring av e-postadresse.',
```

Comment:
stettar -> støttar

Reply:
You sure? https://ordbokene.no/nn/73672

Anyways, I created a PR: https://github.com/AtB-AS/mittatb-app/pull/4152

Reply:
Merged 🤝